src/models/sparse.jl

```diff
@@ -57,3 +57,21 @@ function pullback_evaluate(∂B, tensor::SparseEquivTensor{T}, Rnl, Ylm,
    return ∂Rnl, ∂Ylm
 end
 
+
+_nl(bb) = [(n = b.n, l = b.l) for b in bb]
+
+function get_nl(tensor::SparseEquivTensor{T}) where {T}
```

Comment:
can we call this `get_nnll_spec` please? Can can you add a short inline documentation also to explain what this returns?

Reply:
The reason I call this `get_nl` is that `ACE1` and `ACEpotentials` calls that too. How about depreciating it?

Reply:
I see - I prefer we don't tie ourselves to bad decisions made in the past :)